testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/model/UserSessionPersisterProviderTest.java

```diff
@@ -43,21 +43,45 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Set;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+
 import org.keycloak.models.Constants;
 import org.keycloak.testsuite.arquillian.annotation.AuthServerContainerExclude.AuthServer;
+import org.keycloak.testsuite.util.ClientBuilder;
+import org.keycloak.testsuite.util.RealmBuilder;
 
 /**
  * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
  */
 @AuthServerContainerExclude(AuthServer.REMOTE)
 public class UserSessionPersisterProviderTest extends AbstractTestRealmKeycloakTest {
 
+    @Override
+    public void addTestRealms(List<RealmRepresentation> testRealms) {
+        super.addTestRealms(testRealms);
+
+        // Add another realm "test2"
+        RealmRepresentation testRealm2 = RealmBuilder.create().name("test2").id("test2")
```

Comment:
The ID in case of map storage implementation needs to be a UUID. Can this be changed accordingly?
Disclaimer: Realm storage is right now being implemented, that's why the map storage tests did not catch this.
cc: @vramik

Reply:
@hmlnarik Thanks for the feedback! In this case, I rather changed this test to use realm names instead of realmId, so that test work for both JPA and mapStorage. Also I removed the method "id" from the RealmBuilder class, which I introduced in this PR (in the first version of this PR before you spot this).